xchange-lakebtc/pom.xml

```diff
@@ -25,6 +25,12 @@
 	<dependencies>
 
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.3.2</version>
+		</dependency>
```

Comment:
@timmolter not sure what version from pom I am suppsed to remove, the file was generated before me, I just added commons-io. 

Yes, I was referring to Travi CI build, the examples module needs the new lakebtc jar in order to compile.